chain/jsonrpc-primitives/src/types/transactions.rs

```diff
@@ -0,0 +1,152 @@
+use near_primitives::borsh::BorshDeserialize;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+#[derive(Debug, Clone)]
+pub struct RpcTransactionRequest {
```

Comment:
I suggest we rename this to `RpcBroadcastTransactionRequest` to avoid confusion (I had to look around to make sure it is indeed used for broadcast endpoints (well, it is also used for `check_tx`, but that is fine, I believe, to use `RpcBroadcastTransactionRequest` there as well)